config/deploy/production.rb

```diff
@@ -11,13 +11,11 @@
 set :branch, ENV['BRANCH'] || 'master'
 
 def zk_host
-  "lib-zk1:2181,lib-zk2:2181,lib-zk3:2181"
+  "lib-zk1:2181,lib-zk2:2181,lib-zk3:2181/solr7"
```

Comment:
Is this using the group membership feature of Zookeeper?

Reply:
This is what zookeeper calls "chroot" and what solr calls a "znode"

Reply:
This is what zookeeper calls "chroot" and what solr calls "znode"